homeassistant/loader.py

```diff
@@ -119,6 +119,11 @@ class BlockedIntegration:
     "variable": BlockedIntegration(
         AwesomeVersion("3.4.4"), "prevents recorder from working"
     ),
+    # Added in 2024.10.1 because of
+    # https://github.com/alandtse/alexa_media_player/issues/2579
+    "alexa_media": BlockedIntegration(
+        AwesomeVersion("4.13.3"), "crashes Home Assistant"
```

Comment:
```suggestion
        AwesomeVersion("4.13.4"), "crashes Home Assistant"
```
This version here represents the  `lowest_good_version`, and should point to the first version fixing the problems, not the version with the problems.
So we need to _at least_ set this to `4.13.4` and hope it is fixed there, or `None`, to block it completely until we release a new version of core with the known fixed version.

Reply:
Problem is that many people rolled back to 4.13.2 which is unaffected, if it does represent lowest_good_version, that would be blocked as well. but I guess that can't be avoided easily without additional changes.